A52-xds-custom-lb-policies.md

```diff
@@ -154,6 +155,10 @@ To provide the `xds_wrr_locality` load balancer information about locality weigh
 
 Since an xDS configuration can place a given locality under multiple priorities, it is possible to see locality weight attributes with different values for the same locality. We do not support this kind of an edge case and just use the weight in the first attribute we encounter. A warning should be logged about any locality weights being discarded.
 
+### CDS Load Balancer
+
+In the `xds_cluster_resolver` config, the `xds_lb_policy` field comes directly from the field of the same name in the CDS update struct. If the `xds_cluster_resolver` config is generated from an aggregate cluster tree, the value of the `xds_lb_policy` field will be determined from the CDS update for the root cluster.
```

Comment:
I don't think we need the second sentence here.  As mentioned above, I think this should be stated in A37, and nothing in A52 actually changes it, so I don't think it needs to be stated here.

Reply:
As I said in the other comment, I don't think it makes sense to say this in A37, and since A42 and A52 can each be implemented without the other, I think it makes sense to make this clarification in both.

Reply:
I don't think it's true that A52 can be implemented without A42.  (It doesn't require the ring_hash policy itself, but it does require the structural changes in A42.)  Without A42, the `xds_cluster_resolver` config still encodes the LB policy config in the split `locality_picking_policy` and `endpoint_picking_policy` fields, and that does not provide a single way to send the combined LB policy config down.

I think A37 is a prereq for A42, and A42 is a prereq for A52.  And neither A42 nor A52 actually change the aggregate cluster behavior described in this text, which was actually introduced in A37.

Reply:
A52 explicitly says that the `xds_cluster_resolver` config has a single `xds_lb_policy` field that is "used as the child config of the xds_cluster_impl LB policy". The only thing there that actually depends on A42 is the part about "Move Load Reporting into xds_cluster_impl Policy", because A52 needs that change for the same reason that A42 needs it.

Reply:
A42 explicitly covers replacing the `locality_picking_policy` and `endpoint_picking_policy` fields with the `xds_lb_policy` field:

https://github.com/grpc/proposal/blob/e045287a90083bd448280c94499e256bd2b12bc4/A42-xds-ring-hash-lb-policy.md#change-child-policy-config-generation-in-xds_cluster_resolver-policy

A52 does not make sense without that.